src/app/pages/schedule-filter/schedule-filter.html

```diff
@@ -19,11 +19,12 @@
   <ion-list [lines]="ios ? 'inset' : 'full'">
     <ion-list-header>Tracks</ion-list-header>
 
-    <ion-item *ngFor="let track of tracks" [attr.track]="track.name | lowercase">
-      <ion-icon *ngIf="ios" slot="start" [name]="track.icon" color="medium"></ion-icon>
-      <ion-label>{{track.name}}</ion-label>
-      <ion-checkbox [(ngModel)]="track.isChecked"></ion-checkbox>
-    </ion-item>
+<ion-item *ngFor="let track of tracks" [attr.track]="track.name | lowercase">
+  <ion-icon *ngIf="ios" slot="start" [name]="track.icon" color="medium"></ion-icon>
+  <ion-checkbox [(ngModel)]="track.isChecked">
+    {{track.name}}
+  </ion-checkbox>
+</ion-item>
```

Comment:
Looks like opening the filter box a second time causes it to use legacy checkboxes again for some reason?

https://user-images.githubusercontent.com/90629384/228283907-44284394-7815-4ee3-a1ce-bd1e06ba2d88.mp4

Reply:
This looks to possibly be a bug with the modern syntax + Angular. I'm not able to get the same behavior with the modern syntax and the React conference app. However the React conference app is using inline modals and the Angular conference app is using controller modals. 

Adding `aria-label` to the `ion-checkbox` does consistently render correctly, but for some reason our automatic detection of a slotted label is not working in Angular (when rendered the second+ time).

e.g.: This "works around" the issue.
```html
<ion-checkbox [(ngModel)]="track.isChecked" [attr.aria-label]="track.name">
   {{ track.name }}
</ion-checkbox>
```

Reply:
I confirmed that this only happens on the new syntax. It is loading from this file: https://github.com/ionic-team/ionic-conference-app/blob/master/src/assets/data/data.json

```typescript
return this.http
  .get('assets/data/data.json')
  .pipe(map(this.processData, this));
```

Reply:
I tried what @sean-perkins mentioned on the call this morning and do not see the same problem with `ion-toggle` so it seems to only affect `ion-checkbox`. 

I added `aria-label` as a temporary workaround and created a ticket to fix this in the framework - [FW-3891](https://ionic-cloud.atlassian.net/browse/FW-3891). I referenced the ticket above in the commit with the workaround - https://github.com/ionic-team/ionic-conference-app/pull/1790/commits/19110292e038b941eb57d093f689e42175cc405f.